### ISSUE TITLE
Use template-literal instead of EJS

### ISSUE THREAD

**Issue by Drulac** [mbret/chewie-system#1]
[Template Literal](https://github.com/Drulac/template-literal) is fastest, smallest and simplest template engine, because it use JS's literal template feature.

It's  55 times faster than EJS, and it also use less CPU and RAM ressources, so it may be a good idea to use it instead of EJS 😀

**Comment by mbret** [comment]
You are completely right and i'm using string literal for a long time now :) The fact is that I don't have any more free time to give to this project :/ I don't even remember the state where I left the app as it's still unfinished. Are you actually using it ?

**Comment by Drulac** [comment]
I don't use it, but I saw that you use EJS, so I take the time to tell it yo you 😊

**Comment by mbret** [comment]
Ah I see, thank you then :)
